lib/staticryptJs.js

```diff
@@ -63,11 +63,17 @@ function init(staticryptConfig, templateConfig) {
      *   expose more information in the future we can do it without breaking the password_template
      */
     async function handleDecryptionOfPage(password, isRememberChecked) {
-        const { isRememberEnabled, rememberDurationInDays, staticryptSaltUniqueVariableName } = staticryptConfig;
-        const { rememberExpirationKey, rememberPassphraseKey } = templateConfig;
+        const { staticryptSaltUniqueVariableName } = staticryptConfig;
 
         // decrypt and replace the whole page
         const hashedPassword = await cryptoEngine.hashPassword(password, staticryptSaltUniqueVariableName);
+        return handleDecryptionOfPageAndRemember(hashedPassword, isRememberChecked)
+    }
+    exports.handleDecryptionOfPage = handleDecryptionOfPage;
+
+    async function handleDecryptionOfPageAndRemember(hashedPassword, isRememberChecked) {
```

Comment:
I think I'd call it `handleDecryptionOfPageFromHash`, since technically `handleDecryptionOfPage` also handles the `isRememberChecked` param and the difference between the two is the hashed password